internal/state/actions.go

```diff
@@ -103,7 +103,7 @@ func (state *State) actionCreateMessage(
 		return 0, err
 	}
 
-	var reqs []*db.CreateMessageReq
+	var reqs db.CreateMessageReq
```

Comment:
Tiny thing, but could you call this "req" since there's just one?

Reply:
Done